src/test/java/org/prebid/server/bidder/adkernel/AdkernelBidderTest.java

```diff
@@ -42,7 +42,7 @@
 
 public class AdkernelBidderTest extends VertxTest {
 
-    private static final String ENDPOINT_URL = "http://%s/hb?zone=%s";
+    private static final String ENDPOINT_URL = "https://pbs.adksrv.com/hb?zone=%s";
```

Comment:
We don't use real endpoints in tests, please, replace with something like `"https://test.com?zone=%s"`